commons/all/src/main/java/org/infinispan/commons/dataconversion/JavaSerializationEncoder.java

```diff
@@ -12,8 +12,8 @@
 @Deprecated
 public class JavaSerializationEncoder extends MarshallerEncoder {
 
-   public JavaSerializationEncoder(ClassWhiteList classWhiteList) {
-      super(new JavaSerializationMarshaller(classWhiteList));
+   public JavaSerializationEncoder(ClassAllowList classAllowList) {
```

Comment:
If we keep `ClassWhiteList` and make it extend `ClassAllowList`, this ctor should remain the same for backwards compatibility and we can just pass it to `JavaSerializationMarshaller`.